library/core/src/ptr/mod.rs

```diff
@@ -2166,10 +2166,10 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
     }
 }
 
-/// Align pointer `p`.
+/// Overalign pointer `p`.
```

Comment:
Hm, while I can see where this is coming from, my fresh eyes today see this summary sentence as incorrect, both in its previous incarnation as well as the new proposed one.

What this function really does is along the lines of


```suggestion
/// Calculate an offset to pointer `p` overaligned to alignment `a`.
```

perhaps? Mind adding this to your PR?